src/validation.cpp

```diff
@@ -191,6 +190,7 @@ class CChainState {
     CBlockIndex* FindMostWorkChain();
     bool ReceivedBlockTransactions(const CBlock &block, CValidationState& state, CBlockIndex *pindexNew, const CDiskBlockPos& pos, const Consensus::Params& consensusParams);
 
+    void PruneBlockIndexCandidates();
```

Comment:
In commit "Move LoadChainTip into CChainState",

Nit, revert this unrelated change?

Reply:
How is this unrelated? Because of the change PruneBlockIndexCandidates (an internal helper function) is no longer called outside of CChainState, so it should be made private.

Reply:
Oh missed the visibility change.